docs/docs/operate/autoscalingstrategies.md

```diff
@@ -0,0 +1,145 @@
+      "rope": [0.0, 0.0],
+      "kp": 0.01,
+      "kd": 0.01
+  },
+  "network": {
+      "setPoint": 60.0,
+      "rope": [25.0, 0.0],
+      "kp": 0.01,
+      "kd": 0.01
+  }
+}
+```
+
+| Field           | Description  |
+| --------------- | ------------ |
+| `minSize`       | Minimum number of workers in the stage. It will not scale down below this number. |
+| `maxSize`       | Maximum number of workers in the stage. It will not scale up above this number. |
+| `cooldownSeconds` | This indicates how many seconds to wait after a scaling operation has been completed before beginning another scaling operation. |
+| `maxAdjustment` | Optional. The maximum number of workers to scale up/down in a single operation. |
+| `rps`           | Expected RPS per worker. Must be > 0. |
+| `cpu`, `memory`, `network` | Configure PID controller for each resource. |
+| `setPoint`      | Target set point for the resource. This is expressed as a percentage of allocated resource to the worker. For example, `60.0` on `network` means network bytes should be 60% of the network limit on machine definition. |
+| `rope`          | Lower and upper buffer around the set point. Metric values within this buffer are assumed to be at set point, and thus contributes an error of 0 to the PID controller. |
+| `kp`            | Multiplier for the proportional term of the PID controller. This will affect the size of scaling actions. |
+| `kd`            | Multiplier for the derivative term of the PID controller. This will affect the size of scaling actions. |
+
+### Clutch RPS
+
+This strategy scales the stage base on number of events processed. The target set point is a percentile of RPS. The signal
+is the sum of RPS, inbound drops, Kafka lag, and outbound drops from source jobs. Therefore, it effectively tries to keep
+drops and lag at 0. It takes the first 10 minutes after job launch to learn the first RPS set point. Afterwards, it may
+adjust the set point once every hour. Set point should become stable the longer a job runs, since it simply takes
+a percentile of historical RPS metric.
+
+The source job drop metric is not enabled by default. It is only applicable if your job connects to an upstream job as
+input. You can enable this metric by setting the job parameter `mantis.jobmaster.autoscale.sourcejob.metric.enabled` to true.
+Further, you need to specify the source job targets in the job parameter `mantis.jobmaster.autoscale.sourcejob.target`.
+You can omit this if your job already has a `target` parameter for connecting to source jobs, the auto scaler will pick
+that up automatically. Example:
+
+```json
+{
+  "targets": [
+    {
+      "sourceJobName": "ConsolidatedLoggingEventKafkaSource"
+    }
+  ]
+}
+```
+
+Optionally, tt is possible to further customize the behavior of the PID controller. You can supply the configuration as
+JSON in the job parameter `mantis.jobmaster.clutch.config`. Example:
+
+```json
+{
+  "rpsConfig": {
+    "setPointPercentile": 0.50,
```

Comment:
we should make these values consistent with other autoscaling policies to avoid confusion when using different policies. Eg. setpoint value is 50.0 in other autoscaling policies, and expressed as 0.5 here

Reply:
This makes sense, I agree we should make it consistent. I will update the documentation first and then put a diff to match the scale change.